pontoon/projects/templates/projects/widgets/project_list.html

```diff
@@ -49,10 +49,14 @@
   </tr>
 {% endmacro %}
 
-{% macro footer(request=False) %}
+{% macro footer(request=False, locale_code='') %}
```

Comment:
A more elegant way of passing main template context to a macro is by using `with context` on `import`:
```jinja
{% import 'projects/widgets/project_list.html' as ProjectList with context %}
```